spec/models/storage_location_spec.rb

```diff
@@ -145,6 +145,26 @@
       end
     end
 
+    describe "item_total_value" do
+      it "returns total vale of a specific item in this storage location" do
+        storage_location = create(:storage_location)
+        item = create(:item, value_in_cents: 500)
+        create(:inventory_item, storage_location_id: storage_location.id, item_id: item.id, quantity: 10)
+        expect(storage_location.item_total_value(item.id)).to eq(50)
+      end
+    end
+
+    describe "inventory_total_value" do
+      it "returns total value of all items in this storage location" do
+        storage_location = create(:storage_location)
+        item1 = create(:item, value_in_cents: 100)
+        item2 = create(:item, value_in_cents: 200)
+        create(:inventory_item, storage_location_id: storage_location.id, item_id: item1.id, quantity: 10)
+        create(:inventory_item, storage_location_id: storage_location.id, item_id: item2.id, quantity: 10)
+        expect(storage_location.inventory_total_value).to eq(30)
```

Comment:
Reading it here, I think the method name should indicate the units it's returning. We go back and forth between cents and dollars, and that seems like ripe territory for introducing a logic bug somewhere, unintentionally. 

Could you change the method name to `inventory_total_value_in_dollars` or `inventory_total_value_in_cents` (whichever direction you choose to go in)?